GTG/gtk/browser/tag_context_menu.py

```diff
@@ -52,7 +52,7 @@ def __build_menu(self):
         if self.tag is not None:
             # Color chooser FIXME: SHOULD BECOME A COLOR PICKER
             self.mi_cc = Gtk.MenuItem()
-            self.mi_cc.set_label(_("Edit Tag..."))
+            self.mi_cc.set_label(_("Edit..."))
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en